tests/tests.js

```diff
@@ -79,9 +79,13 @@ var runTests = function runTests(arrayFrom) {
 
 	test('it includes Object.prototype values when it is polluted', function (t) {
 		/* eslint-disable no-extend-native */
-		Object.prototype[1] = 42;
```

Comment:
Okay, fixed. However, I think Object.prototype customizing has a risk that cause bugs in depended libraries.